websites/C/ComicK/metadata.json

```diff
@@ -15,7 +15,7 @@
 		"en": "🚀 ComicK - Fastest Comic Reader",
 		"vi_VN": "🚀 ComicK - Trang đọc truyện tranh nhanh nhất"
 	},
-	"url": "comick.app",
+	"url": "comick.cc",
```

Comment:
Regex? They seem to own atleast:
.app
.cc
.ink
.fun

Reply:
okay boss 🫡

Reply:
However
![image](https://github.com/PreMiD/Presences/assets/73395916/eabd9b49-2bb5-4b99-bcb2-2449af0ab015)

Reply:
They only seem to use their .cc domain

Reply:
But they still own it, and like all other anime websites will probs change domain again in a few months

Reply:
https://www.reddit.com/r/ComicK/comments/qvcgzj/faq_updates_domains/
>Can't seem to access our site, here are some domains we use
>Main Domain-- https://comick.app
>
>Second Domain - - https://comick.ink

No need for a regex if I provide an array of urls right?

Reply:
> But they still own it, and like all other anime websites will probs change domain again in a few months

yea sure but keep in mind ^